Src/AutoFixtureUnitTest/Kernel/TerminatingWithPathSpecimenBuilderTest.cs

```diff
@@ -162,35 +159,62 @@ public void SpecimenRequestsWillInitiallyBeEmpty()
         [Fact]
         public void SpecimenRequestsRaisedFromTracerAreRecordedCorrectly()
         {
-            // Fixture setup
-            var tracer = new DelegatingTracingBuilder();
-            var specimens = new[] { new object(), new object(), new object() };
-            var requestEvents = specimens.Select((o, i) => new RequestTraceEventArgs(o, i)).ToList();
-            var sut = new TerminatingWithPathSpecimenBuilder(tracer);
-            // Exercise system
-            requestEvents.ForEach(tracer.RaiseSpecimenRequested);
-            // Verify outcome
-            Assert.True(specimens.SequenceEqual(sut.SpecimenRequests));
-            // Teardown
+            // arrange
```

Comment:
Throughout the AutoFixture codebase, the [Four Phase Test](http://xunitpatterns.com/Four%20Phase%20Test.html) naming convention is used. Particularly, for existing tests, I don't see any reason to switch to [Arrange-Act-Assert](http://wiki.c2.com/?ArrangeActAssert)? 😃